utah-design-system-website/src/react/components/websiteContent/library/components/forms/MultiSelect/MultiSelectExampleRender.jsx

```diff
@@ -49,6 +49,8 @@ export function MultiSelectExampleRender({
           });
         })}
         values={values}
+        // @ts-ignore
+        autoComplete="off"
```

Comment:
These autoCompletes were used for testing... should they be left in? Should all examples have autocomplete turned off?